docs/company.md

```diff
@@ -0,0 +1,335 @@
+
+__Company__
+
+`GET /1/companies.json`
+Get a list of companies
+
+* Required permission: read
+* Arguments:
+    + `limit`: amount of results (default: 50)
+    + `page`: page to show (default: 1)
+* Sample response:
+
+````
+HTTP/1.1 200 OK
+{
+  "companies": [
+    {
+      "id": "42",
+      "name": "Magnetis",
+      "description": "Magnetis is the best online advisor ever",
+      "logo": "/companies/42/logo.png",
+      "website": "magnetis.com.br",
+      "social_links": [
+        "twitter.com/@magnetis"
+      ],
+      "markets": [
+        "financial services",
```

Comment:
@lucianot Don't you want to have a table for markets instead? If so, this would be id/index based.